tests/test_app/test_tree.py

```diff
@@ -85,9 +84,6 @@ def test_composable_apps(self):
         )
         self.assertIsInstance(proc, tree_app.quick_tree)
         self.assertIsInstance(proc.input, dist.fast_slow_dist)
```

Comment:
I would prefer using a different name for instance of `fast_slow_dist` class. It is the same as class name.

Reply:
Changed to `calc_dist`